public/static/app/perf-bb/SignalProcessingCtrl.js

```diff
@@ -83,14 +82,14 @@ mciModule.controller('SignalProcessingCtrl', function(
   }, {
     title:    'Unmark',
     action:   _.partial(markFn, PROCESSED_TYPE.NONE),
-    visible:  function() { return state.mode == 'processed' },
+    visible:  function() { return state.mode === 'processed' },
```

Comment:
I guess this also counts as a real change, but I figure you're not looking for a strict check here?